src/main/java/org/influxdb/dto/Point.java

```diff
@@ -269,6 +266,11 @@ void setFields(final Map<String, Object> fields) {
     this.fields = fields;
   }
 
+  public Map<String, Object> getFields() {
```

Comment:
Immutablity the main goal of having builders, you gain guaranteed correct objects and faster synchronization. See: Josh Bloch "effective Jave" e.g.: http://www.informit.com/articles/article.aspx?p=1216151&seqNum=2